.idea/configTemplates/sqldialects.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="SqlDialectMappings">
+    <file url="file://$PROJECT_DIR$/server/modules/platform/audit/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/audit/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/core/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/core/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/experiment/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/experiment/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/issues/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/issues/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/list/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/list/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/mothership/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/pipeline/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/pipeline/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/query/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/query/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/search/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/search/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/study/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/study/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/survey/resources/schemas/dbscripts/postgresql" dialect="PostgreSQL" />
+    <file url="file://$PROJECT_DIR$/server/modules/platform/survey/resources/schemas/dbscripts/sqlserver" dialect="TSQL" />
```

Comment:
Could we have the `ijConfigure` task (or a separate task) set the SQL dialects for all the modules currently checked out?

Reply:
Didn't realize the tasks aren't defined in the gradle plugin. This seems significantly more complicated than the other new tasks.

Reply:
Yeah, to do it for all modules checked out (as opposed to in the settings file) would be considerably slower and since you can set it when you visit a file and then just go about your business without need to do that again, think I'll leave it.